plugins/woocommerce-admin/client/task-lists/setup-task-list/components/task-headers/woocommerce-payments.js

```diff
@@ -54,12 +58,20 @@ const WoocommercePaymentsHeader = ( { task, trackClick } ) => {
 			/>
 			<div className="woocommerce-task-header__contents">
 				<h1>{ __( "It's time to get paid", 'woocommerce' ) }</h1>
-				<p>
-					{ __(
-						"You're only one step away from getting paid. Verify your business details to start managing transactions with WooPayments.",
-						'woocommerce'
-					) }
-				</p>
+				{ incentive?.task_header_content ? (
+					<p
+						dangerouslySetInnerHTML={ sanitizeHTML(
```

Comment:
this `dangerouslySetInnerHTML` scares a bit 🙂 since we're sanitizing the HTML, we should be good XSS-wise, right? no need to stripslash additionally?

Reply:
Yeah, it sounds a bit scary 🙂 You can [read more about it in React docs](https://react.dev/reference/react-dom/components/common#common-props). It's safe to render it after using `sanitizeHTML`, which uses [DOMPurify](https://github.com/cure53/DOMPurify) under the hood.